### ISSUE TITLE
Precompiling Distributions error

### ISSUE THREAD

**Issue by nil-is-lin** [JuliaStats/Distributions.jl#1628]
julia> using Distributions
[ Info: Precompiling Distributions [31c24e10-a181-5473-b8eb-7969acd0382f]
ERROR: LoadError: InitError: could not load library "F:\***\.julia\artifacts\8c1c1529ea289add57db6b790a3039ba491acdce\bin\libRmath-julia.dll"
The specified module could not be found.

 ✗ Rmath
  ✗ StatsFuns
  ✗ Distributions

(@v1.6) pkg> status Distributions
      Status `F:\***\.julia\environments\v1.6\Project.toml`
  [31c24e10] Distributions v0.25.76
Long-term support (LTS) release: v1.6.7 and Current stable release: v1.8.2 have been tried and both have this problem.

**Comment by devmotion** [comment]
That seems to be a problem with Rmath. It seems its binary artifact was not downloaded/installed correctly. So I assume it will fail with the same error message when you try `julia> using Rmath` (possibly you have to install it explicitly first if it's not a direct dependency of your environment yet). I don't know what's the best way to fix such issues, maybe try to remove the artifacts folder in the error message (ie, `"F:***.julia\artifacts\8c1c1529ea289add57db6b790a3039ba491acdce/"`).

**Comment by nil-is-lin** [comment]
Yes, this problem is actually a problem with the Rmath package. If this dependency is gone, will the function of generating Gaussian random numbers still work?
